index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h4>Hello world we are developers</h4>
+
+
+<h1 id="title">Survey form </h1>
+  <p id="description"> Lorem ipsum dolor sit amet consectetur
+  </p>
+
+<form id="survey-form">
+  <label for="name" id="name-label">Name</label>
+    <input id="name" placeholder="name" required>
+    <label for="email" id="email-label">Email</label>
+    <input type="email" id="email" placeholder="email" required>
+    <label for="number" id="number-label">Number</label>
+    <input type="number" id="number" min="7" max="15" required placeholder="number">
+    <select id="dropdown" name="dropdown">
+      <option value="">Number</option>
+      <option> Number</option>
+
+    </select>
+    <br><br>
+    <input type="radio" name="group1" value="option 1">
+    <input type="radio" name="group1" value="option 1">
+    <input type="radio" name="group1" value="option 1">
+    <input type="radio" name="group1" value="option 1">
+    <br><br>
+    <input type="checkbox"value="check1">
+    <input type="checkbox"value="check2">
+    <textarea></textarea>
+
+    <button id="submit" type="submit">Submit</button>
+
+</form>
+
+
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+    -->
```

Comment:
kindly remove commented code. professionally it's good practice not to push commented code.
Also this makes your work clean.

Reply:
kindly check the changes I made from your request